framework/Objects/Meta_Fields.php

```diff
@@ -0,0 +1,266 @@
+<?php
+
+namespace JustCoded\WP\Framework\Objects;
+
+/**
+ * Meta_Fields is the base class for acquiring data from custom fields.
+ */
+
+class Meta_Fields {
+
+	/**
+	 * Plugin used for extending of post custom fields.
+	 *
+	 * Available values:
+	 *      'just-custom-fields'
+	 *      'advanced-custom-fields'
+	 *
+	 * @var string
+	 */
+	public $custom_fields_plugin = 'just-custom-fields';
```

Comment:
however fields cache property is general for both final classes, so it should be in this class